src/types/data/organizationDoc.ts

```diff
@@ -23,6 +23,8 @@ export interface OrganizationDoc {
   brand: {
     primary: string;
     primaryDark?: string;
+    favicon?: string;

```

Comment:
Suggestion: It's a bit unclear what this field needs; maybe something like `faviconUrl`, `faviconSrc` or `faviconHref` would make this more self-documenting.

Reply:
Good call! Done.